Makefile

```diff
@@ -150,7 +151,7 @@ build-image: $(DOCKERFILE_DEVEL)
 		--build-arg MOQ_VERSION="$(MOQ_VERSION)" \
 		--tag $(BUILDIMAGE) \
 		-f $(DOCKERFILE_DEVEL) \
-		.
+		$(K8S_TEST_INFRA)
```

Comment:
This still doesn't allow us to customize the docker image with other dependencies, correct?

Reply:
Correct. The whole purpose of https://github.com/NVIDIA/k8s-dra-driver/pull/83 was to enable local builds with specific versions, but no extra dependencies. 
This PR simply makes sure we continue to hold a central Dockerfile for Build environments, but allows to perform local builds with _custom_ dep versions